weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/sections/ComparisonDefinitionSection/EvaluationDefinition.tsx

```diff
@@ -43,11 +44,10 @@ export const EvaluationModelLink: React.FC<{
 }> = props => {
   const {useObjectVersion} = useWFHooks();
   const evaluationCall = props.state.summary.evaluationCalls[props.callId];
-  const modelObj = props.state.summary.models[evaluationCall.modelRef];
-  const objRef = useMemo(
-    () => parseRef(modelObj.ref) as WeaveObjectRef,
-    [modelObj.ref]
-  );
```

Comment:
@tssweeney  does this make sense? I think we actually never needed `modelObj` because we are only using it for its ref? Which I think is `=` to `evaluationCall.modelRef` but could be wrong...

Reply:
looks correct!